composer.json

```diff
@@ -30,23 +30,24 @@
     "require": {
         "php": "^7.4 || >=8.1",
         "composer-runtime-api": "^2.2",
-        "dealerdirect/phpcodesniffer-composer-installer": "^0.5 || ^0.7",
-        "phpstan/phpstan": "^1.9",
+        "dealerdirect/phpcodesniffer-composer-installer": "^0.5 || ^0.7 || ^1.0",
+        "phpstan/phpstan": "^1.9 || ^1.10",
         "phpstan/phpstan-mockery": "^1.1",
-        "phpstan/phpstan-nette": "^1.1",
-        "phpstan/phpstan-phpunit": "^1.2",
-        "phpstan/phpstan-strict-rules": "^1.4",
-        "slevomat/coding-standard": "^8.5",
+        "phpstan/phpstan-nette": "^1.2",
+        "phpstan/phpstan-phpunit": "^1.3",
+        "phpstan/phpstan-strict-rules": "^1.5",
+        "slevomat/coding-standard": "^8.8",
         "squizlabs/php_codesniffer": "^3.7"
     },
     "require-dev": {
-        "roave/security-advisories": "dev-master",
+        "mockery/mockery": "^1.5",
         "phpunit/phpunit": "^7.5 || ^8.5",
-        "mockery/mockery": "^1.3"
+        "roave/security-advisories": "dev-latest",
+        "symplify/easy-coding-standard": "^11.2.10"
```

Comment:
Shouldnt we include this in require? There is no way to run the checks without it and it seems clunky to require it in the repo that uses these coding standards.